Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -3619,7 +3619,7 @@ public void run() {
 	 * @see BranchError	
 	 */
 	public interface BranchListResponseListener {
-		public void onReceivingResponse(JSONArray list, BranchError error);
+		void onReceivingResponse(JSONArray list, BranchError error);
```

Comment:
I'm curious how you're seeing this from the C# compiler? Or are you just saying the corollary concept in C# doesn't even allow you to do this?